server/render/index.js

```diff
@@ -74,8 +75,40 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
+function getCacheKey( context ) {
+	if ( ! isSectionIsomorphic( context.store.getState() ) || context.user ) {
+		return false;
+	}
+
+	if ( isEmpty( context.query ) ) {
+		return context.pathname;
+	}
+
+	let queryParams = Object.keys( context.query );
+	const cacheConfiguration = get( getIsomorphicSectionConfiguration( context.store.getState() ), 'cache', null );
+
+	if ( cacheConfiguration && cacheConfiguration.queryParams === true ) {
+		return context.canonicalPath;
```

Comment:
Yeah, I don't think this is available. @tug You might need to extend the isomorphic helper's [`getEnhancedContext()`](https://github.com/Automattic/wp-calypso/blob/730ee93301a56f07e44242d9275164576188a2ae/server/isomorphic-routing/index.js#L49-L63).